lib/core/storage.js

```diff
@@ -179,9 +179,25 @@ export default class Storage {
     if (isUnset(value)) {
       Cookies.remove(_key, _options)
     } else {
-      Cookies.set(_key, value, _options)
-    }
 
+      // Support server set cookies
+      if (process.server) {
+        const { parse, serialize } = require('cookie')
```

Comment:
It doesn't make much sense to import only in this branch if it's also imported at the top of the file.

Also, you shouldn't ideally use `require` in a file that is using es modules syntax.

It's probably OK to always (regardless if server or not) import IMO.

Reply:
I also think importing at the top is best. Is the performance boost really worth it? Or should I refactor to only using the already existing import at the top? @pi0 
```
import { parse as parseCookie, serialize as serializeCookie } from 'cookie'
```

Reply:
Counting tree-shaking it saves `serialize` from client bundle. But agree about code readability to move to top import.